lib/components/term.js

```diff
@@ -177,7 +177,20 @@ export default class Term extends Component {
   }
 
   clear() {
-    this.term.onVTKeystroke('\f');
+    // Command Running
+    if (this.term.getCursorColumn() === 0) {
```

Comment:
hmm yeah, I've thought about this, while this seems better than it was before, it's not perfect. I'll have a look 🔍